Install.ps1

```diff
@@ -32,24 +32,54 @@ if ($null -eq (Get-Command py -ErrorAction SilentlyContinue))
     Exit 1
 }
 
-$pythonVersion = (py --version).split(" ")[1]
-if ([version]$pythonVersion -lt [version]"3.7.0")
+$supportedPythonVersions = "3.9", "3.8", "3.7"
+if (Test-Path env:INSTALL_PYTHON_VERSION)
 {
-    Write-Output "Found Python version:" $pythonVersion
-    Write-Output "Installation requires Python 3.7 or later"
-    Exit 1
+    $pythonVersion = $env:INSTALL_PYTHON_VERSION
+}
+else
+{
+    foreach ($version in $supportedPythonVersions)
+    {
+        try
+        {
+            py -$version --version 2>&1 >$null
+            $result = $?
+        }
+        catch
+        {
+            $result = $false
+        }
+        if ($result)
+        {
+            $pythonVersion = $version
+            break
+        }
+    }
+
+    if (-not $pythonVersion)
+    {
+        $reversedPythonVersions = $supportedPythonVersions.clone()
+        [array]::Reverse($reversedPythonVersions)
+        $reversedPythonVersions = $reversedPythonVersions -join ", "
+        Write-Output "No usable Python version found, supported versions are: $reversedPythonVersions"
+        Exit 1
+    }
 }
-Write-Output "Python version is:" $pythonVersion
 
-$openSSLVersionStr = (py -c 'import ssl; print(ssl.OPENSSL_VERSION)')
-$openSSLVersion = (py -c 'import ssl; print(ssl.OPENSSL_VERSION_NUMBER)')
+$fullPythonVersion = (py -$pythonVersion --version).split(" ")[1]
+
+Write-Output "Python version is:" $fullPythonVersion
```

Comment:
These are the best nits since they are easy and non-controversial...  but...  also wrong.  Because PowerShell apparently outputs multiple parameters each on a separate line.  So, how about this?

```suggestion
Write-Output "Python version is: $fullPythonVersion"
```